src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -436,42 +436,37 @@ public function getEditorAjax($filePath, $line)
      * act as an Ajax request. The editor must be a
      * valid callable function/closure
      *
-     * @throws UnexpectedValueException  If editor resolver does not return a boolean
-     * @param  string                   $filePath
-     * @param  int                      $line
-     * @return mixed
+     * @param  string $filePath
+     * @param  int    $line
+     * @return array
      */
     protected function getEditor($filePath, $line)
     {
-        if ($this->editor === null && !is_string($this->editor) && !is_callable($this->editor))
-        {
-            return false;
+        if (!$this->editor || (!is_string($this->editor) && !is_callable($this->editor))) {
+            return [];
         }
-        else if(is_string($this->editor) && isset($this->editors[$this->editor]) && !is_callable($this->editors[$this->editor]))
-        {
+
+        if (is_string($this->editor) && isset($this->editors[$this->editor]) && !is_callable($this->editors[$this->editor])) {
            return [
                 'ajax' => false,
-                'url' => $this->editors[$this->editor],
+                'url'  => $this->editors[$this->editor],
```

Comment:
Usually you don't want to add styling changes to other peoples' code. In some cases it might be difficult to distinguish a style difference from a clear mistake. In this case, the decision to align the values in the array, is purely a stylistic choice. In Whoops we decided not to. Personally, I believe our way is _clearly, objectively_ better, but that's the tricky part with style. :)

But, even more importantly, you don't want to submit these style changes together with other changes in a PR. The developers might want to accept your other changes, but not like the style changes. And that is why every change belongs in its own PR - to allow discussing it individually.